doc/book/in-depth-guide/first-module.md

```diff
@@ -146,19 +146,29 @@ public function getConfig()
 Reload your application and you'll see that everything remains as it was. Next
 we add the new route to our configuration file:
 
+<<<<<<< bb6a2c65745fbea17b98e2e100e676bbcab45a1b
```

Comment:
You've not got your resolutions done!

(My first clue was the `~~~~` markers, as those have been gone since yesterday.)

If you look at the current code (see https://github.com/zendframework/tutorials/blob/bb6a2c65745fbea17b98e2e100e676bbcab45a1b/doc/book/in-depth-guide/first-module.md), all invokable declarations are already updated to factories, and pointing to `InvokableFactory::class` already.

As such, I'm going to close. Fetch the latest master and comb through that anew if you feel inclined. :smile: